src/main/java/com/xceptance/neodymium/util/SelenideAddons.java

```diff
@@ -160,7 +160,8 @@ public ElementsCollection get()
             }
             catch (final Throwable t)
             {
-                if (isThrowableCausedBy(t, StaleElementReferenceException.class))
+                if (isThrowableCausedBy(t, StaleElementReferenceException.class)
+                    || t.getMessage().contains("StaleElementReferenceException"))
```

Comment:
could add the String "StaleElementReferenceException" as a private static final field and create it out of the Java class to be able to detect future changes.
something like `StaleElementReferenceException.class.getName()`

Reply:
Done